plugins/modules/wafv2_web_acl.py

```diff
@@ -370,6 +446,7 @@ def main():
         cloudwatch_metrics=dict(type='bool', default=True),
         metric_name=dict(type='str'),
         tags=dict(type='dict'),
+        custom_response_bodies=dict(type='dict', default=dict()),
```

Comment:
```suggestion
        custom_response_bodies=dict(type='dict'),
```

A default value make the detection more complex